.github/workflows/interop-tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Interoperability tests with GnuTLS and NSS
+on: [pull_request, push]
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      matrix:
+        COMPONENT: [openssl-upstream-gnutls, openssl-upstream-nss]
+    env:
+      COMPONENT: ${{ matrix.COMPONENT }}
+    # Code below inspired by https://fedoramagazine.org/github-actions-use-podman-to-run-fedora-linux/
+    steps:
+      - name: Setup Docker
```

Comment:
What's the point of installing docker? GitHub runners have it preinstalled?

Reply:
I'll drop that, I'm just trying to get a built container image that will build openssl and incorporate the test plans right now